scripts/generate_function_json.py

```diff
@@ -0,0 +1,66 @@
+import json
+from pathlib import Path
+from argparse import ArgumentParser
+from subprocess import check_output, CalledProcessError, PIPE
+
+parser = ArgumentParser()
+parser.add_argument('--source', required=True)
+parser.add_argument('--binary', required=True)
+args = parser.parse_args()
+
+
+def get_result(example: str) -> str:
+    try:
+        out = check_output(
+            [
+                args.binary,
+                '-json',
+                '-c',
+                'SELECT setseed(0.42); ',
+                '-c',
+                'LOAD icu;',
+                '-c',
+                f'SELECT {example.strip(";")} AS result'
+            ],
+            stderr=PIPE
+        )
+        rows = json.loads(out.splitlines()[-1])
+        return rows[0]['result']
+    except CalledProcessError as e:
+        print(example.strip(), e.stderr.decode())
+        return None
+
+
+def main():
+    functions = []
+    source = Path(args.source)
+    assert source.exists()
+    for file in source.glob('src/core_functions/**/*.json'):
```

Comment:
The only improvement I could see here is that eventually we might want to go full on with duckfooding, where we use duckdb itself (either from within python or as a bash script) to generate functions.json. 🦆